### DISCUSSION TITLE
2024-02-11-Release-Lite-Stable-Bookworm-arm64

### DISCUSSION THREAD

**Discussion by mgrouch** [bareboat-necessities/lysmarine_gen#374] [General]
BBN Marine Linux OS for Raspberry Pi
2024-02-11-Release-Lite-Stable-Bookworm-arm64
Stable Release based on Bookworm (Long Term Support)

64-bit, Bookworm, supports raspberry pi 4/400/CM4 and raspberry pi 5 (aka rpi5)

This is the lite version.

Due to many users complaining about bloating BBN OS with too many packages
this release has been slimmed down to more basic ones.

<hr /><em>This discussion was created from the release <a href='https://github.com/bareboat-necessities/lysmarine_gen/releases/tag/v2024-02-11'>2024-02-11-Release-Lite-Stable-Bookworm-arm64</a>.</em>

**Comment by mgrouch** [comment]
A video of one of the boat refit projects using it

https://youtu.be/tRMc2yQZEDc?feature=shared

**Reply by gildas-declercq** [reply]
Hi, 
is there a doc with comparison between lite and full version ?
thanks a lot

**Reply by mgrouch** [reply]
There is forum post which explains it. 

https://github.com/bareboat-necessities/lysmarine_gen/discussions/361
